frontends/ol-components/src/components/Button/Button.tsx

```diff
@@ -127,6 +127,15 @@ const ButtonStyled = styled.button<ButtonStyleProps>((props) => {
       // Pill-shaped buttons... Overlapping border radius get clipped to pill.
       borderRadius: "100vh",
     },
+    edge === "none" && {
+      border: "none",
+      ":hover:not(:disabled)": {
+        "&&": {
+          backgroundColor: "inherit",
+        },
+      },
```

Comment:
What's this for? It seems surprising to me. I wouldn't expect `edge` to affect the background color.

- With this style, some variants hover strangely with `edge="none"` (fill, tertiary)
- Without this...the new cards seem to look OK? and buttons seem to hover OK?

Reply:
The idea is that when edge is "none" we're on mobile, so no hover state. It does happen though that if you are on desktop and shrink your window, the hover area doesn't look good.

<img width="122" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/08be7c7b-4a37-4fe3-a621-335c48462f95">

I'm anticipating this will be removed from here / updated in favor of the Card Action Buttons, though we don;t seem to have a variant of these for the edge-less mobile variant https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=4026-61418&m=dev

<img width="455" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/cc453531-4b9e-4fbc-8f04-acd304d705ab">

@steven-hatch ?